admin_guide/pruning_resources.adoc

```diff
@@ -278,6 +288,20 @@ image streams that have a reference to the image in `*status.tags*`.
 operation is not allowed.
 ====
 
+[NOTE]
+====
+Separating the removal of {product-title} Image API Objects and Image data
+from the Registry by using `--prune-registry=false` followed by
+xref:#hard-pruning-registry[Hard Pruning the Registry] narrows some timing windows,
+and is safer when compared to trying to prune both through one command. However,
+timing windows are not completely removed.
+
+For example, you can still create a Pod referencing an Image as pruning identifies that
+Image for pruning. You should still keep track of an API Object created during the pruning
+operations that might reference Images, so you can mitigate any references to deleted
+content.
```

Comment:
+1, but @legionus may have some additional comments on what we should say here.

Reply:
Please add a warning that re-doing the pruning without `--prune-registry` option or with `--prune-registry=true` will not lead to pruning the storage of image registry. I mean that all images that was pruned with `--prune-registry=false` can be deleted from the registry storage only by using the hard-pruning otherwise the space will be lost.

Reply:
Good catch!